# One Address

uint8 PHONE_TYPE_HOME=0
uint8 PHONE_TYPE_WORK=1
uint8 PHONE_TYPE_MOBILE=2

# string first_name
# string last_name
# string phone_number
# uint8 phone_type

# An Address Book
rosidl_tutorials_msgs/Contact[] address_book